support/mock/mockrt.go

```diff
@@ -263,14 +264,37 @@ func (rt *Runtime) GetRandomness(tag crypto.DomainSeparationTag, epoch abi.Chain
 			"         requested epoch: %d greater than current epoch %d\n", epoch, rt.epoch)
 	}
 
-	exp := rt.expectRandomness[0]
+	exp := rt.expectRandomnessBeacon[0]
 	if tag != exp.tag || epoch != exp.epoch || !bytes.Equal(entropy, exp.entropy) {
 		rt.failTest("unexpected get randomness\n"+
 			"         tag: %d, epoch: %d, entropy: %v\n"+
 			"expected tag: %d, epoch: %d, entropy: %v", tag, epoch, entropy, exp.tag, exp.epoch, exp.entropy)
 	}
 	defer func() {
-		rt.expectRandomness = rt.expectRandomness[1:]
+		rt.expectRandomnessBeacon = rt.expectRandomnessBeacon[1:]
+	}()
+	return exp.out
+}
```

Comment:
hey @nikkolasg, can you verify that this is correct according to the spec: https://github.com/filecoin-project/specs/pull/967 ?